lookyloo/modules/vt.py

```diff
@@ -33,7 +33,7 @@ def module_init(self) -> bool:
             self.logger.info('Not enabled')
             return False
 
-        self.client = vt.Client(self.config['apikey'])
+        self.client = vt.Client(self.config['apikey'], trustenv=bool(self.config['trustenv'], False))
```

Comment:
just FYI, the correct name of the parameter is `trust_env`, not `trustenv`. And `bool(self.config['trustenv'], False)` isn't a valid syntax. I guess you wanted to do something like `self.config.get('trustenv', False)`

Reply:
:scream: so sorry! Thanks for the catch!

Reply:
No worries, it is fixed in the repo ;)